src/main/resources/CU-ApplicationResources.properties

```diff
@@ -691,3 +691,6 @@ error.document.contractsGrantsInvoice.prorate.no.award.budget.total=The budget t
 pdp.send.ach.notification.error.report.file.prefix=pdpSendAchAdviceNotificationsErrorReport
 pdp.send.ach.notification.error.report.title=PDP Send ACH Advice Notification Error Report
 pdp.send.ach.notification.error.report.line.format=%25s %25s %25s %55s
+
+validationPatternRegex.integer=-?[0-9]+
+error.format.edu.cornell.kfs.kns.datadictionary.validation.fieldlevel.IntegerValidationPattern=The {0} must be an integer.
```

Comment:
Will "[0]" be a field name or a value?  If it is a field name, I think "the" can be eliminated.  The error doesn't read all that well.

Reply:
The placeholder is for a field name. I'll remove the extra "the" to make it easier to read. Thanks.